es-spatial-plugin/src/cmr/elasticsearch/plugins/spatial/script/core.clj

```diff
@@ -28,23 +30,57 @@
 ;;                         Begin script helper functions                     ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(def parameters
+  "The parameters to the Spatial script"
+  [:ords :ords-info :operator])
+
+(defn- extract-params
+  "Extracts the parameters from the params map given in the script."
+  [script-params]
+  (when script-params
+    (into {} (for [param parameters]
+               [param (XContentMapValues/nodeStringValue
+                        (get script-params (name param)) nil)]))))
+
 (defn- get-from-fields
   [^LeafStoredFieldsLookup lookup key]
   (when (and lookup key (.containsKey lookup key))
     (when-let [^FieldLookup field-lookup (.get lookup key)]
       (seq (.getValues field-lookup)))))
 
+(defn any-true?
+  "Returns true if predicate f returns a truthy value against any of the items.
+  This is very similar to some but it's faster through it's use of reduce."
+  [f items]
+  (some true?
+    (map f items)))
+
+(defn every-true?
+  "Returns true if predicate f returns a truthy value against all of the items."
+  [f items]
+  (every? true?
+    (map f items)))
+
+(defn remove-br
+  "Removes bounding rectangles from granule spatial data."
+  [shapes]
+  (let [shapes-no-br (into [] (remove #((instance? cmr.spatial.mbr.Mbr (first shapes)) shapes)))]
```

Comment:
I don't think this line is correct. Shouldn't this be `(remove #(instance? cmr.spatial.mbr.Mbr (first %))`?

Reply:
Fixed.